src/diffusers/loaders.py

```diff
@@ -684,6 +686,11 @@ def load_textual_inversion(
             token (`str` or `List[str]`, *optional*):
                 Override the token to use for the textual inversion weights. If `pretrained_model_name_or_path` is a
                 list, then `token` must also be a list of equal length.
+            text_encoder ([`~transformers.CLIPTextModel`], *optional*):
+                Frozen text-encoder ([clip-vit-large-patch14](https://huggingface.co/openai/clip-vit-large-patch14)).
+                If not specified, function will take self.tokenizer.
+            tokenizer ([`~transformers.CLIPTokenizer`], *optional*):
+                A `CLIPTokenizer` to tokenize text. If not specified, function will take self.tokenizer.
```

Comment:
So, only single text encoder and tokenizer for now? I'd imagine you'd need to touch both the text encoders to squeeze out performance from SDXL. But, I could very well be wrong.

Reply:
Ah nevermind. It was made clear in the snippet.